contracts/handlers/ERCHandlerHelpers.sol

```diff
@@ -24,6 +26,14 @@ contract ERCHandlerHelpers is IERCHandler {
     // token contract address => is burnable
     mapping (address => bool) public _burnList;
 
+    // token contract address => decimals
+    mapping (address => Decimals) public _decimals;
```

Comment:
Could we perhaps pack it together with contractWhitelist/burnList ? Or do it as a separate PR? Could leave the getters for backwards compatibility.

Reply:
Yep, I have a separate PR in the works for this